crates/reporting/src/error/canonicalize.rs

```diff
@@ -2522,6 +2548,18 @@ fn pretty_runtime_error<'b>(
 
             title = "OPTIONAL FIELD IN RECORD BUILDER";
         }
+        RuntimeError::ReturnOutsideOfFunction(region) => {
+            doc = alloc.stack([
+                alloc.concat([
+                    alloc.reflow("The "),
+                    alloc.keyword("return"),
+                    alloc.reflow(" keyword can only be used in functions."),
+                ]),
+                alloc.region(lines.convert_region(region), severity),
+            ]);
```

Comment:
Similar issue here, the top line should be more of a fact than a directive, like:
```
This `return` appears in a top-level definition:

    return 42

However, they are only allowed inside functions.
```

Reply:
That error wasn't actually being used, so I removed it.